docs/umi/isoseq-bcstats.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+parent: Single cell
+title: Barcode Frequency Statistics via bcstats
```

Comment:
This is very long and you'll see how it will wrap 4x in the menu bar
<img width="261" alt="image" src="https://user-images.githubusercontent.com/1433286/168151243-121a6c6b-7c98-4a85-9534-7b381e29c9de.png">

Reply:
Trimming to "Barcode Statistics".